docs/ecosystem-center/publish-other-ecosystems.md

```
@@ -1,4 +1,4 @@
- ## 如何发布其他生态
+ # 如何发布其他生态
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)